extensions/resteasy-reactive/rest-client-jaxrs/deployment/src/main/java/io/quarkus/jaxrs/client/reactive/deployment/RestClientRemoveTrailingSlashBuildItem.java

```diff
@@ -0,0 +1,9 @@
+package io.quarkus.jaxrs.client.reactive.deployment;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * This build item configures all the REST Clients to remove the trailing slash from the paths.
+ */
+public final class RestClientRemoveTrailingSlashBuildItem extends MultiBuildItem {
```

Comment:
What's the usecase to force all clients to do this? 

Shouldn't this be a per-client setting?

Reply:
I tried to do the same as for disable smart default produces, which is also done for all the clients. 
But will update the PR to do this per-client instead +1

Reply:
🙏🏽